src/python/pants/notes/2.13.x.md

```diff
@@ -366,12 +366,6 @@
 
 * Add options to turn off `tailor` on a per-language basis ([#15335](https://github.com/pantsbuild/pants/pull/15335))
 
-* Add lz4 support to archive extraction ([#15245](https://github.com/pantsbuild/pants/pull/15245))
```

Comment:
Several changes from 2.12.0a0 were included here. I suspect that the changelog script was run against the latest dev release, not a0.